### ISSUE TITLE
it is not possible to enter Arabic or Persian numbers into input

### ISSUE THREAD

**Issue by samranfeli** [bl00mber/react-phone-input-2#459]
its not possible to enter numbers when keyboard language is Arabic or Persian:
٠١٢٣٤٥٦٧٨٩

**Comment by kimixz** [comment]
I have the same problem

**Comment by bl00mber** [comment]
I may look to support Eastern Arabic and some few other numeral systems. You may try to make PR yourself. For now you can only use function in your app that will convert Eastern Arabic numerals to Western Arabic and back.

**Comment by archonkulis** [comment]
@bl00mber Hey! Any tips on how to achieve that? Inside onChange I can't catch the Arabic symbols (or any other letters) as it accepts only numbers. Much appreciated.

**Comment by bl00mber** [comment]
@archonkulis you are right the validator is probably a blocker so you can not use these numerals right now.

**Comment by Siraj2025** [comment]
@bl00mber Is this issue still present or has it been resolved?
